Scripts/DemoApp-StreamDevelopers.xcscheme

```diff
@@ -59,7 +59,7 @@
       </EnvironmentVariables>
    </LaunchAction>
    <ProfileAction
-      buildConfiguration = "Release"
+      buildConfiguration = "Profile"
```

Comment:
That is the important change. All the other are just about renaming `ReleaseTests` to `Profile`